ofl/adventpro/DESCRIPTION.en_us.html

```diff
@@ -4,6 +4,9 @@
     give an edge to your typography.
 </p>
 <p>
-    To contribute to the project, please see <a
-        href="https://github.com/googlefonts/Advent">github.com/googlefonts/Advent</a>.
-</p>
\ No newline at end of file
+    The family was upgraded to a variable font with additional Italic styles in December 2023.
```

Comment:
Shouldn’t this say “December 2022” instead? =)

Reply:
Was corrected in previous commit 3 days ago https://github.com/google/fonts/pull/5522/commits/4e7a3bf5849bda3cafdc713623d310bbb95375b6